Assets/Scripts/PlayerController/PlayerMovement.cs

```diff
@@ -28,6 +28,9 @@ public class PlayerMovement : MonoBehaviour
     private float m_JumpHeight = 1.3f;
 
     private CharacterController m_CharacterController;
+    private TextChanger m_textChanger;
```

Comment:
Why do you need text changer in here? Don't we already use this in `PowerupsManager`?

Reply:
Yikes didn't notice! Will fix in next commit!